curation-authors-backfill/index.spec.ts

```diff
@@ -0,0 +1,241 @@
+import { handlerFn } from './';
+import * as SecretManager from './secretManager';
+import * as Jwt from './jwt';
+import sinon from 'sinon';
+import nock from 'nock';
+import config from './config';
+import { SQSEvent } from 'aws-lambda';
+import * as ProspectApi from './externalCaller/prospectApiCaller';
+
+// async function stubInsertCuratedItem() {
+//   return;
+// }
+
+describe('curation migration', () => {
+  const record = {
+    externalId: '123',
+    url: 'https://www.test.com/test-story',
+    title: 'Equine dentist wins lottery',
+    publisher: 'The Snort & Tail',
+  };
+
+  beforeEach(() => {
+    // mock the secrets manager call
+    sinon
+      .stub(SecretManager, 'getCurationToolsDataSyncPrivateKey')
+      .resolves('test-secret');
+
+    // mock the generate jwt function
+    sinon.stub(Jwt, 'generateJwt').returns('test-jwt');
+    // sinon
+    //   .stub(curatedItemIdMapper, 'insertCuratedItem')
+    //   .callsFake(stubInsertCuratedItem);
+  });
+
+  afterEach(() => {
+    sinon.restore();
+    jest.clearAllMocks();
+  });
+
+  describe('lambda handler', () => {
+    xit('returns batch item failure if prospect-api has error, with partial success', async () => {
+      nock(config.AdminApi)
+        .post('/') //prospect-api call for first event
+        .reply(200, {
+          data: {
+            getUrlMetadata: {
+              isSyndicated: true,
+              isCollection: false,
+              publisher: 'Gums Weekly',
+            },
+          },
+        })
+        .post('/') //curated-corpus-api call for first event
+        .reply(200, {
+          data: {
+            importApprovedCorpusItem: {
+              approvedItem: {
+                externalId: 'random-approvedItem-guid',
+              },
+              scheduledItem: {
+                externalId: 'random-scheduledItem-guid',
+                scheduledSurfaceGuid: 'new_tab_en_us',
+              },
+            },
+          },
+        })
```

Comment:
this should be modified to our authors backfill scenario, yeah?

Reply:
I've updated them all and there's no need to be skipping any of those tests any more.